plugins/discovery-gce/build.gradle

```diff
@@ -7,20 +7,30 @@ esplugin {
 }
 
 versions << [
-  'google': '1.23.0'
+  'google'              : '1.41.4',
```

Comment:
Sorry for the delay on this, I'm not sure if these versions align with what the library expects? 
```
[INFO] \- com.google.apis:google-api-services-compute:jar:v1-rev235-1.25.0:compile
[INFO]    \- com.google.api-client:google-api-client:jar:1.25.0:compile
[INFO]       +- com.google.oauth-client:google-oauth-client:jar:1.25.0:compile
[INFO]       |  +- com.google.http-client:google-http-client:jar:1.25.0:compile
[INFO]       |  |  +- org.apache.httpcomponents:httpclient:jar:4.5.5:compile
[INFO]       |  |  |  +- org.apache.httpcomponents:httpcore:jar:4.4.9:compile
[INFO]       |  |  |  +- commons-logging:commons-logging:jar:1.2:compile
[INFO]       |  |  |  \- commons-codec:commons-codec:jar:1.10:compile
[INFO]       |  |  \- com.google.j2objc:j2objc-annotations:jar:1.1:compile
[INFO]       |  \- com.google.code.findbugs:jsr305:jar:3.0.2:compile
[INFO]       +- com.google.http-client:google-http-client-jackson2:jar:1.25.0:compile
[INFO]       |  \- com.fasterxml.jackson.core:jackson-core:jar:2.9.6:compile
[INFO]       \- com.google.guava:guava:jar:20.0:compile
```
Am I missing something? @arteam

Reply:
You're correct! Let me pinpoint the exact versions of the transitive dependencies instead of the latest.

Reply:
It seems like they discontinued that library in favor of https://mvnrepository.com/artifact/com.google.cloud/google-cloud-compute/1.8.1?

Reply:
Interesting! Let me try this dependency

Reply:
It seems it still gets updated! https://repo1.maven.org/maven2/com/google/apis/google-api-services-compute/v1-rev20220322-1.32.1/ got released a couple of days go. It seems that the artefacts stopped being indexed by mvnrepository.com at some point. I should have used https://search.maven.org/search?q=google-api-services-compute from the beggining :facepalm:

Reply:
Okay, it's there https://mvnrepository.com/artifact/com.google.apis/google-api-services-compute/v1-rev20220322-1.32.1, just the sorting on https://mvnrepository.com/artifact/com.google.apis/google-api-services-compute is broken

Reply:
Oh 🤦, I was looking into the google cloud documentation and I didn't manage to find a good release list... thanks for looking into this!